projects/ad7606b_fmc/common/ad7606b_bd.tcl

```diff
@@ -0,0 +1,83 @@
+# data, read and write lines
+
+create_bd_port -dir O -from 15 -to 0 rx_db_o
+create_bd_port -dir I -from 15 -to 0 rx_db_i
+create_bd_port -dir O rx_db_t
+create_bd_port -dir O rx_rd_n
+create_bd_port -dir O rx_wr_n
+
+# control lines
+
+create_bd_port -dir I rx_busy
+create_bd_port -dir O rx_cnvst_n
+create_bd_port -dir O rx_cs_n
+create_bd_port -dir I rx_first_data
+
+# instantiation
+
+ad_ip_instance axi_ad7606b axi_ad7606b
+ad_ip_parameter axi_ad7606b CONFIG.IF_TYPE 1
+ad_ip_parameter axi_ad7606b CONFIG.EXTERNAL_CLK 0
+
+ad_ip_instance axi_pwm_gen axi_pwm_gen
+ad_ip_parameter axi_pwm_gen CONFIG.ASYNC_CLK_EN 0
+ad_ip_parameter axi_pwm_gen CONFIG.N_PWMS 1
+ad_ip_parameter axi_pwm_gen CONFIG.PULSE_0_WIDTH 135
+ad_ip_parameter axi_pwm_gen CONFIG.PULSE_0_PERIOD 136 
+
+ad_ip_instance axi_dmac axi_ad7606b_dma
+ad_ip_parameter axi_ad7606b_dma CONFIG.DMA_TYPE_SRC 2
+ad_ip_parameter axi_ad7606b_dma CONFIG.DMA_TYPE_DEST 0
+ad_ip_parameter axi_ad7606b_dma CONFIG.CYCLIC 0
+ad_ip_parameter axi_ad7606b_dma CONFIG.DMA_2D_TRANSFER 0
+ad_ip_parameter axi_ad7606b_dma CONFIG.DMA_DATA_WIDTH_SRC 16
```

Comment:
For 8 channels 16bits per channel, the pack core out bus is 128 bits, DMA_DATA_WIDTH_SRC should be the same.